components/raftstore/src/store/config.rs

```diff
@@ -152,6 +152,21 @@ pub struct Config {
     pub lock_cf_compact_interval: ReadableDuration,
     pub lock_cf_compact_bytes_threshold: ReadableSize,
 
+    /// The duration of time to wait before attempt scheduling a full
+    /// compaction. If not set, periodic full compaction is not run
+    /// automatically.
+    ///
+    /// NOTE: This feature is highly experimental!
+    pub periodic_full_compact_tick_interval: ReadableDuration,
```

Comment:
I think periodic_full_compact_tick_interval can be the internal implementation and not necessarily to be exposed in user's configuration. Because once periodic_full_compact_start_times is specified, TiKV theoretically knows better how often to check if needs to schedule a full compact check.
For example, when tikv get started and have  with "23:00, 03:00" and the start time is 21:00. Then tikv knows the next tick can be scheduled at 2 hour later.

Reply:
ok, how does this sound:
1. i will make it a constant (30 minutes).
2. unless start times are specified, do not run full compaction at all.

Reply:
done.